### PR TITLE
Fixing checkstyle errors in getting-started-java (whitespace and import order).

### PR DESCRIPTION
**Pull request by dzlier-gcp**
Part of the effort to have an all-green [kokoro dashboard](https://fusion.corp.google.com/).

### CODE REVIEW

File: appengine-standard-java8/springboot-appengine-standard/src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2018 Google Inc.
```
Comment (kurtisvg) [inline appengine-standard-java8/springboot-appengine-standard/src/main/resources/application.properties]: This is supposed to be "Google LLC" now (this just started recently).

File: helloworld-springboot/src/main/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2018 Google Inc.
```
Comment (lesv) [inline helloworld-springboot/src/main/resources/application.properties]: Google LLC

File: appengine-standard-java8/springboot-appengine-standard/src/main/resources/application.properties (hunk 1 above)
Comment (dzlier-gcp) [reply appengine-standard-java8/springboot-appengine-standard/src/main/resources/application.properties]: Done.

File: helloworld-springboot/src/main/resources/application.properties (hunk 2 above)
Comment (lesv) [reply helloworld-springboot/src/main/resources/application.properties]: missed this one.

**Comment by lesv** [comment]
So you didn't get the one I called out, and only the 2018 (ie. new notices should be LLC), the older copyrights were (are) correct as Inc.

**Pull request merged**